src/api/firebase.js

```diff
@@ -55,7 +55,7 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	const listCollectionRef = collection(db, listId);
 	// TODO: Replace this call to console.log with the appropriate
 	// Firebase function, so this information is sent to your database!
-	return console.log(listCollectionRef, {
+	return await addDoc(listCollectionRef, {
```

Comment:
Same comment as the other PR - handling errors if addDoc fails. See https://github.com/the-collab-lab/tcl-56-smart-shopping-list/pull/18/files#diff-d274a54187c91ba0f532df2a9e194e27ab50e988f5e4c33f5a7893918320c661R35